### PR TITLE
draft post on CoC comm call

### PR DESCRIPTION
**Pull request by stefaniebutland**

### CODE REVIEW

File: _drafts/2016-12-21-commcallv12-review-coc.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+3. reporting and enforcing
+
+## Drafting a Code of Conduct
+
+Once a group of interested people digs into the work of drafting a CoC, the process can seem like it's going on forever, especially if you try to incorporate "everyone's" feedback. The CoC cannot be a living document. It's most important to produce something good and perhaps revisit it each year. There seemed to be a consensus that a small committee of key stakeholders should draft the CoC. For his own lab, Titus drafted and posted it without fanfare, but has already seen [unanticipated benefits](http://ivory.idyll.org/blog/2016-notes-on-lab-coc.html).
+
+Some notes:
+
+* Develop and post a CoC whether or not you know of any problems; setting clear expectations is 90% of the work
+* The CoC has to set expectations for everyone, including the boss
```
Comment (sckott) [inline _drafts/2016-12-21-commcallv12-review-coc.md]: maybe "including the boss (if there is one)"

File: _drafts/2016-12-21-commcallv12-review-coc.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  - blog
+tags:
+- ropensci
+- community
+- events
+
+---
+Our [Community Call](https://ropensci.org/blog/blog/2016/10/31/comm-call-v12) on December 15th covered a big topic in tech communities: "How do I create a code of conduct for my event/lab/codebase?". Here, we cover some of the key themes and considerations that arose from the discussion and point to curated resources and examples to follow when developing a code of conduct (CoC) for your community.
+
+Three guest speakers shared different perspectives. Dr Pauline Barmby talked about the process and lessons learned as Data and Software Carpentries recently updated their CoC; Ms Safia Abdalla talked about "Codes of conduct for open source: the stuff no one tells you"; and Dr Titus Brown talked about his lab CoC. You can [read their bios](#speaker-bios) below and watch the [video](https://vimeo.com/196503807) of the call. Speakers did not use other presentation materials.
```
Comment (sckott) [inline _drafts/2016-12-21-commcallv12-review-coc.md]: Maybe define what "Carpentries" is, (SWC and DC)

File: _drafts/2016-12-21-commcallv12-review-coc.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+## Drafting a Code of Conduct
+
+Once a group of interested people digs into the work of drafting a CoC, the process can seem like it's going on forever, especially if you try to incorporate "everyone's" feedback. The CoC cannot be a living document. It's most important to produce something good and perhaps revisit it each year. There seemed to be a consensus that a small committee of key stakeholders should draft the CoC. For his own lab, Titus drafted and posted it without fanfare, but has already seen [unanticipated benefits](http://ivory.idyll.org/blog/2016-notes-on-lab-coc.html).
+
+Some notes:
+
+* Develop and post a CoC whether or not you know of any problems; setting clear expectations is 90% of the work
+* The CoC has to set expectations for everyone, including the boss
+* One big issue for the Carpentries was whether or not there should be specific lists of prohibited behaviors or protected groups
+* Because many of our communities cross international boundaries, our CoCs should be accessible in multiple languages. The [Contributor Covenant](http://contributor-covenant.org/) does provide a template CoC in [multiple languages](http://contributor-covenant.org/i18n/).
```
Comment (sckott) [inline _drafts/2016-12-21-commcallv12-review-coc.md]: one point to include might be that rules differ by country, something to keep in mind

File: _drafts/2016-12-21-commcallv12-review-coc.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+## We Have Work To Do
+rOpenSci currently has a [Code of Conduct for our events](http://unconf16.ropensci.org/coc.html) and for [software onboarding](https://github.com/ropensci/onboarding/blob/master/policies.md/#code-of-conduct). We'll take what we learned in this Community Call to develop a more comprehensive CoC, make sure it's easily findable and that consequences of violations are enforceable. [Let us know](https://ropensci.org/contact.html) if you'd like to get involved.
```
Comment (sckott) [inline _drafts/2016-12-21-commcallv12-review-coc.md]: Could you also include a link to an example CoC that we use for each package?  https://github.com/ropensci/taxize/blob/master/CONDUCT.md  - may also want to mention perhaps not all our pkgs have one, but we're working on it

File: _drafts/2016-12-21-commcallv12-review-coc.md (hunk 1 above)
Comment (stefaniebutland) [inline _drafts/2016-12-21-commcallv12-review-coc.md]: added

File: _drafts/2016-12-21-commcallv12-review-coc.md (hunk 4 above)
Comment (stefaniebutland) [inline _drafts/2016-12-21-commcallv12-review-coc.md]: added

Comment (stefaniebutland) [inline _drafts/2016-12-21-commcallv12-review-coc.md]: thanks for mentioning that one - good to show! 
I was thinking of it and looking but couldn't find cause didn't remember it applied to individual packages! Didn't remember enough to ask about it. When I re-found the software onboarding CoC, thought that was it

File: _drafts/2016-12-21-commcallv12-review-coc.md (hunk 3 above)
Comment (stefaniebutland) [inline _drafts/2016-12-21-commcallv12-review-coc.md]: added

**Pull request merged**